net-analyzer/greenbone-feed-sync/greenbone-feed-sync-24.1.1.ebuild

```diff
@@ -35,6 +35,47 @@ RDEPEND="
 
 distutils_enable_tests unittest
 
+src_test() {
+	# Tests need to change the DEFAULT_CONFIG_FILE path
+	for TARGET in ${PYTHON_TARGETS}
+	do
+		mkdir -p "${WORKDIR}/${P}-${TARGET}/install/etc/gvm" || die
+		chmod +r "${WORKDIR}/${P}-${TARGET}/install/etc/gvm"
```

Comment:
It seems that I can completely remove the for cycle. Even if the default configuration file is not really available, to avoid the sandbox's violation, it is sufficient to be patch the config.py file.